packages/meditrak-server/src/routes/exportSurveyResponses/exportSurveyResponses.js

```diff
@@ -82,6 +83,9 @@ export async function exportSurveyResponses(req, res) {
   try {
     let country;
     let entities;
+
+    req.assertPermissions(allowNoPermissions);
+
```

Comment:
Exporting survey response already check if users have access to the survey at line 163:
```
      const hasSurveyAccess = accessPolicy.allows(country.code, permissionGroup.name);
```

I'm not sure if it's worth just pulling this 1 line of code to a new permissions checker file. So I kept it as it is and wrote tests at endpoint level to test if the permissions check work when exporting survey responses

Reply:
We could pull this to a new permissions checker if it is to be used in other places too. Till then, I think the above is fine!